packages/pyyaml/patches/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- downloads/pyyaml-5.3.1/Makefile	2020-03-18 21:09:19.000000000 +0000
++++ patches/Makefile	2020-11-20 06:33:24.117490628 +0000
+@@ -1,7 +1,7 @@
+ 
+ .PHONY: default build buildext force forceext install installext test testext dist clean
+ 
+-PYTHON=/usr/bin/python
++PYTHON=/usr/local/bin/python3
```

Comment:
Any way to use,
```
/bin/env python
```
to autodetect the python used?

Reply:
> Thanks @daoxian ! Generally LGTM, however I would rather merge #1102 first which includes some major changes in the build. That should happen over the next few days, after which we should be able to include this PR.

That's great and thank you.